### ISSUE TITLE
Final Status Update: colors has moved to @colors/colors

### ISSUE THREAD

**Issue by DABH** [Marak/colors.js#340]
```
tldr: colors -> @colors/colors
```

Development of the `colors` package has ceased, and further work on the project will take place via the new official repository https://github.com/DABH/colors.js.  The corresponding official package on NPM is [@colors/colors](https://www.npmjs.com/package/@colors/colors).

Please update your package.json to refer to `@colors/colors` instead of `colors`.  Collaborators will be added to the colors org in NPM in due course after interviews are conducted; the very clear intent here is to avoid a situation in the future similar to what happened with this old repo.  In the meantime, PRs are welcome on the new repository.  I am unsubscribing from all threads in this old repo and will not see any notifications here.

Thanks all for your patience over the last several weeks, and I look forward to continuing development and supporting users of `@colors/colors`, plus with the help of some new collaborators soon.

**Comment by rilysh** [comment]
...

**Comment by rnag** [comment]
..

**Comment by gowthamvbhat** [comment]
Feels like you are hijacking the repo users

**Comment by rnag** [comment]
the content was perhaps passable, but the delivery was **not great** -- i'd go so far as to peg it as kinda rude to the content owner; but that's just my two cents, anyway.

**Comment by rufw91** [comment]
Yea this is some BS. Marak needs some $$$$ so you steal his sh*t?

**Comment by moosefaceee** [comment]
I don't understand @rufw91 can you steal **open source** code?

**Comment by gowthamvbhat** [comment]
@moosefaceee It's the community, not the code.

**Comment by PieterjanDeClippel** [comment]
> Yea this is some BS. Marak needs some $$$$ so you steal his sh*t?

He doesn't need money, he needs narcotics. That's why he's so desperate to the extent that he sabotages his own projects.

No employer in the world is going to hire him ever again now that he decided to randomly sabotage software...
